ironfish-rust/src/frost_utils/account_keys.rs

```diff
@@ -37,8 +37,10 @@ pub fn derive_account_keys(
         SaplingKey::new(*group_secret_key).expect("failed to derive group secret key");
 
     // Authorization key (ak), obtained from the multisig setup process
-    let authorizing_key = Option::from(SubgroupPoint::from_bytes(&authorizing_key.serialize()))
-        .expect("failied to derive authorizing key");
+    let mut bytes: [u8; 32] = [0; 32];
+    bytes.copy_from_slice(&authorizing_key.serialize().unwrap());
```

Comment:
We should re-factor this to return a result instead of using an unwrap and expect on the next line